components/style/core/rtl.less

```diff
@@ -0,0 +1,8 @@
+/* stylelint-disable at-rule-no-unknown */
+
+.rtl {
+  direction: rtl;
+}
+.ltr {
+  direction: ltr;
+}
```

Comment:
Do we need global styles now?

Reply:
Maybe not, i'll remove it.

Reply:
These styles will be needed on some cases like direction of `.ant-col` that i'm used.
so every col wrapped in `.rtl` will be displayed correctly. (Used in ConfigProvider RTL demo)

Reply:
@afc163 , I removed `rtl` global styles and added prefixed class to every component. like `ant-cascader-picker-rtl`
Also i fixed grid system direction with another approach.